index.js

```diff
@@ -141,6 +141,14 @@ class SlackNotifier extends NotificationBase {
             message = `${message}\n${metaMessage}`;
         }
 
+        const metaVar = `build.meta.notification.slack.${buildData.jobName}.message`;
+
+        const buildMessage = hoek.reach(buildData, metaVar, { default: false });
+
+        // Job specific slack message takes precedence of generic slack message.
+        if (buildMessage) {
+            message = `${message}\n${buildMessage}`;
```

Comment:
If both generic slack message and specific slack message are set, this message might be `${message}\n${metaMessage}\n${buildMessage}` ?
If specific slack message overwrite generic slack message, should generic slack message not be included this message?

Reply:
Sorry ... you are right. Fixed now.

Reply:
Sorry. Fixed now.